src-docs/src/views/html_id_generator/html_id_generator_reuse.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { EuiFlexGroup, EuiCode } from '../../../../src/components';
+
+import { htmlIdGenerator } from '../../../../src/services';
+
+const generateId = htmlIdGenerator('test');
+
+export const HtmlIdGeneratorReuse = () => {
+  return (
+    <EuiFlexGroup gutterSize="none" alignItems="flexStart" direction="column">
+      <EuiCode>{generateId()}</EuiCode>
+      <EuiCode>{generateId()}</EuiCode>
+      <EuiCode>{generateId()}</EuiCode>
+      <EuiCode>{generateId('hello')}</EuiCode>
+      <EuiCode>{generateId('world')}</EuiCode>
+    </EuiFlexGroup>
```

Comment:
More nits about flex groups. They truly are unnecessary for column layouts especially when there's no guttersize and not use EuiFlexItem within.
```suggestion
    <>
      <EuiCode>{generateId()}</EuiCode><br />
      <EuiCode>{generateId()}</EuiCode><br />
      <EuiCode>{generateId()}</EuiCode><br />
      <EuiCode>{generateId('hello')}</EuiCode><br />
      <EuiCode>{generateId('world')}</EuiCode>
    <>
```

Reply:
👍 Nice, thanks, I'll get less lazy about EuiFlex in the future! Adding this is in separate local commit, as Prettier has Opinions™️ about the formatting

Reply:
https://github.com/elastic/eui/pull/5133/commits/5a5b1670443a914d92158999be06ce7c776709c6